src/platform/backends/shared/linux/process_spec.cpp

```diff
@@ -38,3 +39,9 @@ QString mp::ProcessSpec::working_directory() const
 {
     return QString();
 }
+
+// Set what multipass logging level the stderr of the child process should have
+mpl::Level mp::ProcessSpec::error_log_level() const
+{
+    return mpl::Level::error;
```

Comment:
I think we can default to `warning` here, wdyt?

Reply:
I really don't have a strong opinion. I thought sensible default for a process printing to stderr was to record it as an error. I'm happy to change it. Any other opinions?

Reply:
The problem is `error` gets printed to the user's console, so defaulting to that may be too verbose on the user.

Reply:
Changed to warning. 

Note I kept existing behaviour of dnsmasq logging stderr as "error" which I presume was intentional